lib/kubernetes/client.rb

```diff
@@ -76,13 +76,14 @@ def sync_kubernetes_services
   def any_lb_services_modified?
     k8s_svc_raw = kubectl("get service --all-namespaces --field-selector spec.type=LoadBalancer -ojson")
     svc_list = JSON.parse(k8s_svc_raw)["items"]
+    lb = @load_balancer.reload
```

Comment:
```suggestion
    @load_balancer.reload
```

`reload` updates the object and returns the receiver, it doesn't return a different object. That would allow you to avoid needing to change the following lines. Local variables are faster than instance variables, but with the JIT, the performance difference is minimal (maybe worth it for a loop, probably not for 3 calls).